tests/store_creation/test_creation_boto3store.py

```diff
@@ -7,7 +7,7 @@
 
 storage = pytest.importorskip("google.cloud.storage")
 
-S3_URL = "s3://minio:miniostorage@127.0.0.1:9000/bucketname?create_if_missing=true&is_secure=false"
+S3_URL = "s3://minio:miniostorage@127.0.0.1:9000/bucketname?create_if_missing=true&is_secure=false&verify=False"
```

Comment:
Why is the F uppecase here?

Reply:
No particular reason. It makes no difference, but I've lowercawed it for consistency.